zone/client.h

```diff
@@ -212,6 +212,15 @@ enum eInnateSkill {
 	InnateDisabled = 255
 };
 
+enum MerchantBucketComparison {
```

Comment:
this should be an `enum class` rather than plain enum. (Eventually I would like to switch all enums over, unless they really don't make sense to)